pkg/gateway/gateway.go

```diff
@@ -687,6 +695,18 @@ func (n *jfsObjects) GetObjectInfo(ctx context.Context, bucket, object string, o
 			return minio.ObjectInfo{}, errno
 		}
 	}
+	objMeta, err := n.getObjMeta(n.path(bucket, object))
+	if err != nil {
+		return minio.ObjectInfo{}, err
+	}
+	if opts.UserDefined == nil {
+		opts.UserDefined = make(map[string]string)
+	}
+	for k, v := range objMeta {
+		opts.UserDefined[k] = v
+	}
+	contentType := utils.GuessMimeType(object)
+	contentType = n.getObjContentType(objMeta, contentType)
```

Comment:
The code here is strange, the function `getObjContentType` is only used here, it doesn't need to be a separate function.

Reply:
Inlined.